### ISSUE TITLE
Investigate "Failed to connect to bus" errors in workstation provisioning

### ISSUE THREAD

**Issue by emkll** [freedomofpress/securedrop-workstation#256]
When running `make all` in dom0, there are many errors that state:
```
Failed to connect to bus: no such file or directory
Failed to connect to bus: no such file or directory
Failed to connect to bus: no such file or directory
[...]
```

We should investigate these errors/warnings and consider suppressing them.

**Comment by eloquence** [comment]
This is a very old issue and I don't believe I've observed this issue myself; added "needs repro" for now, we may close if no additional reports in the near future.

**Comment by conorsch** [comment]
Looks potentially related to #625 (which itself needs reproducing). Locally, I'm able to produce a similar message:

```
[user@dom0 ~]$ sudo /srv/salt/update-xfce-settings reset-power-management
update-xfce-settings: Resetting power management options for user root
Failed to init libxfconf: Failed to connect to socket /run/user/0/bus: No such file or directory.
[user@dom0 ~]$ sudo su
[root@dom0 user]# /srv/salt/update-xfce-settings reset-power-management
update-xfce-settings: Resetting power management options for user root
Failed to init libxfconf: Failed to connect to socket /run/user/0/bus: No such file or directory.
```

**Comment by conorsch** [comment]
> Looks potentially related to #625

Nope, those changes didn't exist when this issue was opened. New hypothesis, @emkll is using i3, and that's a correlation. I use xfce and the error is not familiar to me. Quite likely the errors have appeared and I've overlooked them, of course. @zenmonkeykstop are you also using i3, and if so, can you comment on whether this is a reproducible error for you?

**Comment by emkll** [comment]
I've tried a provisioning run in XFCE, and still see these messages in the output (It could still be due to i3 being installed). I will report back once try again on a fresh install

**Comment by rmol** [comment]
Is this simply because it's getting run with `sudo`, so the path contains the wrong user ID?

**Comment by conorsch** [comment]
@rmol Perhaps, although depends on what "it" is that's being run. Maybe adding `-x` to the provision-all script would shed more light, for those who can reproduce it.

**Comment by eloquence** [comment]
Closing as stale, please reopen if you see this again / have seen it recently.
